subjects/listat/README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Instructions
 
-Write a function `ListAt` that takes a pointer to the list `l` and an `int pos` as parameters. This function should return the `NodeL` in the position `pos` of the linked list `l`.
+Write a function `ListAt` that takes a pointer to the head of the list `l` and an `int pos` as parameters. This function should return the `NodeL` in the position `pos` of the linked list `l`.
```

Comment:
This part still remains a contradiction:
> This function should return the `NodeL` in the position `pos` of the linked list `l`

when compared with the return type of the function.
https://github.com/01-edu/public/blob/849cdbf71f1391b9dc23b9be1492ee37c9624ea8/subjects/listat/README.md?plain=1#L18

Reply:
Thanks. I'll address that.